### PR TITLE
feat: add github docs/pr template

### PR DESCRIPTION
**Pull request by afrodynamic**

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Code of Conduct
```
Comment (ika-ika-ika) [inline .github/CODE_OF_CONDUCT.md:1]: Please use version from Google Drive

**Review by ika-ika-ika — CHANGES_REQUESTED** [review]
Please review comments at code_of_conduct.md

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (afrodynamic) [reply .github/CODE_OF_CONDUCT.md:1]: Acknowledged

**Review by ika-ika-ika — APPROVED** [review]
Great job, Harold!

**Review by lederius — APPROVED** [review]
good

**Pull request merged**